src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -70,7 +83,34 @@ def get_config():
 
 def get_cover_file_path():
   """Return location of coverage file for Syzkaller."""
-  return os.path.join(get_work_dir(), 'coverfile')
+  file_path = os.path.join(get_work_dir(), 'coverfile')
+  if not os.path.isfile(file_path):
+    open(file_path, 'w+').close()
+
+  logs.log(f'Rawcover file will be stored in {file_path}')
+  return file_path
+
+
+def find_syzkaller_port(pid: int) -> int:
+  """Find localhost port where syzkaller is connected."""
```

Comment:
Can you import psutil here instead? This code may get imported in environments where psutil isn't available.

Reply:
will do